flutter/lib/src/sentry_flutter_options.dart

```diff
@@ -385,11 +396,33 @@ class SentryFlutterOptions extends SentryOptions {
 class _SentryFlutterExperimentalOptions {
   /// Replay recording configuration.
   final replay = SentryReplayOptions();
-}
 
-/// Callback being executed in [ScreenshotEventProcessor], deciding if a
-/// screenshot should be recorded and attached.
-typedef BeforeScreenshotCallback = FutureOr<bool> Function(
-  SentryEvent event, {
-  Hint? hint,
-});
+  /// Privacy configuration for masking sensitive data in screenshots and Session Replay.
+  /// Screen content masking redaction is:
```

Comment:
nit: shouldn't we use masking exclusively? seems like we're inconsistent with terminology across the codebase

Reply:
I think `masking` is the preferred terminology or what do you mean @vaind?

Reply:
but this is not really blocking this pr, just wanna make sure we use the right term in the future

Reply:
@martinhaintz feel free to merge when you are ready

Reply:
I will review the changes, and update the wording from `redacting` to `masking` or similar.